assets/css/settings.scss

```diff
@@ -299,6 +299,21 @@
 			}
 		}
 	}
+	.sensei-learners-extra {
+		.postbox {
+			padding: 1em 0px 0px 1em;
+			h2 {
+				padding: 0px !important;
+			}
+			.inside {
+				padding: 0px;
+			}
+		}
+		.button-primary[disabled] {
+			background-color: #037ABAFF !important;
```

Comment:
The color codes seem incorrect. They are 7 characters instead of 6.

Reply:
Oops :p Copied it from phpstrom's color picker, it probably is appending the alpha in it as well. Updating it 👍